### PR TITLE
Hugo in docker

### PR DESCRIPTION
**Pull request by jessitron**
This alleviates the need for anyone to install or upgrade hugo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,31 @@
...
+ that docker-compose started up: `docker exec ado-hugo_hugo_1 hugo new episode/my-episode-name.md` *Note - it is highly recommended that you follow this pattern rather than duplicating an existing file, as this will ensure you get the latest and greatest required frontmatter elements*
+
+### option 3: Install hugo
+1.  Install [Hugo](http://gohugo.io) - *NOTE: Please make sure it is at least version 0.58.3*
 2. Clone this repo
-3. If you want to fire up a local copy to see your changes as you go, enter this command `hugo server -w --baseUrl="http://localhost:1313"`
-4. You probably want to do the previous thing in another pane/window, as it needs to keep running. The -w watches for any changes and rebuilds on the fly. You have to set the baseUrl or it will build it with `www.arresteddevops.com`.
-5. If you want to make a new episode, type in `hugo new episode/my-episode-name.md` *Note - it is highly recommended that you follow this pattern rather than duplicating an existing file, as this will ensure you get the latest and greatest required frontmatter elements*
+3. If you want to fire up a local copy to see your changes as you go: `hugo server`
```
Comment (mattstratton) [inline README.md]: Why was this command changed?

Comment (jessitron) [reply README.md]: the docker-compose sets an env var for the base URL, so you don't have to set it yourself.
(this was in the hugo image. not my bright idea, but useful)

Comment (mattstratton) [reply README.md]: If you’re not using docker you have to keep this command as is. That’s what I am asking about - why it’s changed for the “not docker” section.

Comment (jessitron) [reply README.md]: oh shoot. yeah you're right, it should be the same for that

**Pull request merged**